### PR TITLE
Make contact page mobile bg image visible up to 1024px width #90

### PR DESCRIPTION
**Pull request by himanshukrmr**
## Description
Done the changes for 768-1024px, now the mobile background is showing upto 1024px.
pr is for issue number #90

## Related Issue

Resolves:

### CODE REVIEW

**Review by chadstewart — CHANGES_REQUESTED** [review]
Thanks so much for the contribution! Really appreciate taking the time to work on this. Just a quick change and everything should be good to go.

Thanks so much!

File: tailwind.config.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ module.exports = {
       screens: {
         xs: "425px",
         // => @media (min-width: 425px) { ... }
-
+        'lg':'1025px',
```
[the remark this answers is not in the document]
Comment (himanshukrmr) [reply tailwind.config.js:17]: @chadstewart if we remove this code then the mobile background will come from 768px to 1023px is it ok?

[the remark this answers is not in the document]
Comment (chadstewart) [reply tailwind.config.js:17]: > @chadstewart if we remove this code then the mobile background will be shown from 768px to 1023px is it ok?

That's okay. Shouldn't be a problem. The main idea is to get the image to display as soon as the mobile design is displayed.

[the remark this answers is not in the document]
Comment (chadstewart) [reply tailwind.config.js:17]: > @chadstewart if we remove this code then the mobile background will be shown from 768px to 1023px is it ok?

@himanshukrmr That's okay. Shouldn't be a problem. The main idea is to get the image to display as soon as the mobile design is displayed.

**Pull request merged**